src/web/actions/index.js

```diff
@@ -57,6 +55,42 @@ export const flowEditorRedo = createAction('FLOWS/EDITOR/REDO')
 
 export const setDiagramAction = createAction('FLOWS/FLOW/SET_ACTION')
 
+// Content
+export const receiveContentCategories = createAction('CONTENT/CATEGORIES/RECEIVE')
```

Comment:
One thing that at least needs TODO is to handle errors somehow for all these fetches. Or is think doing it for us?

Reply:
App-wide API error handling is to be done